packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -82,15 +82,6 @@ const testKey2 = "another test key";
 const testValue = "test value";
 const testIncrementValue = 5;
 
-const getPendingStateWithoutClose = (container: IContainerExperimental): string => {
-	const containerClose = container.close;
-	container.close = (message) => assert(message === undefined);
-	const pendingState = container.closeAndGetPendingLocalState?.();
```

Comment:
i would keep this function, and just change which ensures closed is not called with an error. just modify this code to be 
```
return container.closeAndGetPendingLocalState?.().then((pendingState)=>{
	assert(typeof pendingState === "string");
	container.close = containerClose;
	return pendingState;
})
```

Reply:
If I go back and use closeAndGet.. all tests that uses getPendingOps() with send === true go stale at line 113 ensureSynchronized because we are disconnected. Not entirely sure what we could do for those tests. Currently they work because getPendingLocalState is not being disconnected but probably I shouldn't have used that method in the first place.

Reply:
Those tests are checking that we are not reapplying stashed ops that were previously sent. Given that now we are disconnected while getting pending ops, I believe those tests don't apply to closeAndGetPendingLocalState anymore.

Reply:
> Given that now we are disconnected while getting pending ops, I believe those tests don't apply to closeAndGetPendingLocalState anymore

Pending ops may still have been successful, so I don't think it's different from synchronously closing when getting pending ops in this regard. I don't see a problem using getPendingLocalState() though, since it covers basically the same thing (recognizing and not resubmitting successful ops).